metadata-ingestion/sink_overview.md

```diff
@@ -0,0 +1,37 @@
+# Sinks
+
+Sinks are **destinations for metadata**.
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/sources-sinks.png"/>
+</p>
+
+In general, the sink will be defined in the recipe after the _source_ like below.
+
+```yaml
+source: ...
+
+sink:
+  type: <sink_type>
+  config: ...
+```
+
+## Types of Sink
+
+When configuring ingestion for DataHub, you're likely to be sending the metadata to DataHub over either one of the following.
+
+- [REST (datahub-rest)](sink_docs/datahub.md#datahub-rest)
+- [Kafka (datahub-kafka)](sink_docs/datahub.md#datahub-kafka)
+- [File](sink_docs/file.md)
```

Comment:
we should separate the file/console sinks and make it clear that they're primarily for debugging